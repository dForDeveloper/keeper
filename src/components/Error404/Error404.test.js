import React from 'react'
import { shallow } from 'enzyme'

describe('Error404', () => {
  let wrapper

})