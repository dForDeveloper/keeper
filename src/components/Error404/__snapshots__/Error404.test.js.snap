// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error404 should match the snapshot 1`] = `
<div
  className="Error404"
>
  <h2
    className="Error404--h2"
  >
    No Note Found
  </h2>
  <Link
    className="Error404--Link"
    replace={false}
    to="/"
  >
    Back to Notes
  </Link>
</div>
`;
